examples/single_calculations/example_mm.py

```diff
@@ -130,7 +130,7 @@ def example_mm(cp2k_code):
         sys.exit(3)
 
     # Check if callgraph is there.
-    if "runtime.callgraph" in calc['retrieved']._repository.list_object_names():  # pylint: disable=protected-access
+    if "runtime.callgraph" in calc['retrieved'].list_object_names():
```

Comment:
I believe, if this change goes first - CI won't be failing, right?

Reply:
right - but you also won't know whether it works, since it won't be tested without the other change

Reply:
Maybe I am missing something, but it looks to me this change won't get unnoticed: the example file will be run, and if `calc['retrieved']._repository` won't work - the example will fail.

Reply:
> Maybe I am missing something, but it looks to me this change won't get unnoticed: the example file will be run, and if calc['retrieved']._repository won't work - the example will fail.

As explained in the description of the PR, the examples were not being run because the old pytest version was not parsing the `pyproject.toml` file.

Reply:
right, make sense then!